lib/get-success-comment.js

```diff
@@ -7,11 +7,11 @@ module.exports = (issueOrMergeRequest, releaseInfos, nextRelease) =>
     nextRelease.version
   } :tada:${
     releaseInfos.length > 0
-      ? `\n\nThe release is available on${
+      ? `\n\nThe release is available on${decodeURIComponent(
```

Comment:
I think we should rather fix this at https://github.com/semantic-release/gitlab/blob/master/lib/publish.js#L121:

```js
   const releaseUrl = urlJoin(gitlabUrl, repoId, `/-/releases/${encodedGitTag}`);
```

Reply:
Thanks. Good hint. Will check tomorrow.

Reply:
Decoding the final release URL now. Should not affect any dependent functionality.